docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -18,90 +37,143 @@ To complete this tutorial:
 
 Install and run {{< param "PRODUCT_NAME" >}} on Linux.
 
 1. [Install {{< param "PRODUCT_NAME" >}}][Linux Install].
-
+{{< docs/ignore >}}
+1. To view the Alloy UI within the sandbox, Alloy must run on all interfaces. Run the following command before you start the alloy service:
+   ```bash
+   sed -i -e 's/CUSTOM_ARGS=""/CUSTOM_ARGS="--server.http.listen-addr=0.0.0.0:12345"/' /etc/default/alloy
+   ```
+{{< /docs/ignore >}}
 1. [Run {{< param "PRODUCT_NAME" >}}][Run on Linux].
 
+<!-- INTERACTIVE ignore START -->
 ### macOS
 
-Install  and run {{< param "PRODUCT_NAME" >}} on macOS.
+Install and run {{< param "PRODUCT_NAME" >}} on macOS.
 
 1. [Install {{< param "PRODUCT_NAME" >}}][macOS Install].
 
 1. [Run {{< param "PRODUCT_NAME" >}}][Run on macOS].
+<!-- INTERACTIVE ignore END -->
+
+You should now be able to access the {{< param "PRODUCT_NAME" >}} UI at [http://localhost:12345](http://localhost:12345).
+
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
 
 ## Set up a local Grafana instance
 
 In this tutorial, you configure {{< param "PRODUCT_NAME" >}} to collect logs from your local machine and send them to Loki.
 You can use the following Docker Compose file to set up a local Grafana instance.
 This Docker Compose file includes Loki and Prometheus configured as data sources.
 
-```yaml
-version: '3'
-services:
-  loki:
-    image: grafana/loki:3.0.0
-    ports:
-      - "3100:3100"
-    command: -config.file=/etc/loki/local-config.yaml
-  prometheus:
-    image: prom/prometheus:v2.47.0
-    command:
-      - --web.enable-remote-write-receiver
-      - --config.file=/etc/prometheus/prometheus.yml
-    ports:
-      - "9090:9090"
-  grafana:
-    environment:
-      - GF_PATHS_PROVISIONING=/etc/grafana/provisioning
-      - GF_AUTH_ANONYMOUS_ENABLED=true
-      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
-    entrypoint:
-      - sh
-      - -euc
-      - |
-        mkdir -p /etc/grafana/provisioning/datasources
-        cat <<EOF > /etc/grafana/provisioning/datasources/ds.yaml
-        apiVersion: 1
-        datasources:
-        - name: Loki
-          type: loki
-          access: proxy
-          orgId: 1
-          url: http://loki:3100
-          basicAuth: false
-          isDefault: false
-          version: 1
-          editable: false
-        - name: Prometheus
-          type: prometheus
-          orgId: 1
-          url: http://prometheus:9090
-          basicAuth: false
-          isDefault: true
-          version: 1
-          editable: false
-        EOF
-        /run.sh
-    image: grafana/grafana:11.0.0
-    ports:
-      - "3000:3000"
-```
+{{< docs/ignore >}}
+> The interactive sandbox has a VSCode like editor that allows you to access files and folders. To access this feature, click on the `Editor` tab. Note that the editor also has a terminal that you can use to run commands. Since some commands assume you are within a specific directory our recommendation is to run the commands in `tab1`.
```

Comment:
```suggestion
> The interactive sandbox has a VSCode-like editor that allows you to access files and folders. To access this feature, click on the `Editor` tab. The editor also has a terminal that you can use to run commands. Since some commands assume you are within a specific directory, we recommend running the commands in `tab1`.
```